enterprise/frontend/src/embedding-sdk/components/public/StaticDashboard/StaticDashboard.tsx

```diff
@@ -0,0 +1,68 @@
+import {
+  DEFAULT_EMBED_DISPLAY_OPTIONS,
+  useDashboardFullscreen,
+  useDashboardRefreshPeriod,
+  useEmbedTheme,
+  useRefreshDashboard,
+} from "metabase/dashboard/hooks";
+import type {
+  EmbedDisplayParams,
+  RefreshPeriod,
+} from "metabase/dashboard/types";
+import { PublicDashboard } from "metabase/public/containers/PublicDashboard/PublicDashboard";
+import { Box } from "metabase/ui";
+import type { DashboardId, ParameterId } from "metabase-types/api";
+
+export const StaticDashboard = ({
+  dashboardId,
+  parameterValues,
+  refreshPeriod: initialRefreshPeriod = null,
+  displayOptions = {},
+}: {
+  dashboardId: DashboardId;
+  parameterValues: Record<ParameterId, string | string[] | null | undefined>;
+  refreshPeriod?: RefreshPeriod;
+  displayOptions?: Partial<EmbedDisplayParams>;
+}) => {
+  const options: EmbedDisplayParams = {
+    ...DEFAULT_EMBED_DISPLAY_OPTIONS,
+    ...displayOptions,
+  };
+
+  const { refreshDashboard } = useRefreshDashboard({
+    dashboardId,
+    queryParams: parameterValues,
+  });
+  const { isFullscreen, onFullscreenChange, ref } = useDashboardFullscreen();
+  const { onRefreshPeriodChange, refreshPeriod, setRefreshElapsedHook } =
+    useDashboardRefreshPeriod({
+      onRefresh: refreshDashboard,
+      initialRefreshPeriod,
+    });
+
+  const { hasNightModeToggle, isNightMode, onNightModeChange, theme } =
+    useEmbedTheme(options.theme);
+
+  return (
+    <Box ref={ref} style={{ overflow: "auto" }}>
```

Comment:
I've added a `Box` here so we can use `useDashboardFullscreen`'s ability to set the fullscreen root to the dashboard, so when people click the "fullscreen" button on an embedded dashboard, only the dashboard will be fullscreen